Dockerfile

```diff
@@ -50,11 +51,15 @@ RUN git clone --depth 1 --branch version_114 https://github.com/WebAssembly/bina
 RUN useradd -ms /bin/bash builduser
 USER builduser
 
-# Install Rust stable
+# Install Rust (using ./rust-toolchain version)
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y \
-    --default-toolchain stable \
```

Comment:
Wouldn't it be easier (and a smaller diff) if you just COPY the rust-toolchain before this step and use `$(cat rust-toolchain)` in place of `stable`?

Reply:
Then you can add rm rust-toolchain to the end with:

```
    --target wasm32-unknown-unknown \
    && rm rust-toolchain
```

Since, immediately after this command, it is no longer useful.

Reply:
I tried a bunch of different things and ended up with my original code, but you're right, it can be simplified. I've pushed a new commit. Thanks for the feedback!